unison-cli/src/Unison/Codebase/Editor/Output.hs

```diff
@@ -153,6 +153,7 @@ data Output
   | NoLastRunResult
   | SaveTermNameConflict Name
   | SearchTermsNotFound [HQ.HashQualified Name]
+  | SearchTermsNotFound' Bool [HQ.HashQualified Name] [HQ.HashQualified Name]
```

Comment:
Some docs on this and how it's different from `SearchTermsNotFound` would be really great.

I would personally prefer a more descriptive name than `'` so we can see at a glance what the differentiator is 😄 

Also a note on what the `Bool` is, and what the difference between the two HQ args is would be nice 😄